tree_unique_args/src/function_inlining.egg

```diff
@@ -0,0 +1,16 @@
+(ruleset function-inlining)
+
+(rule
+    (
+        (= call (Call f arg))
+        (Function f out)
+    )
+    (
+        (let new-id (Id (i64-fresh!)))
+        (union
+            call
+            ; (NewLet new-id arg out)
```

Comment:
Why is NewLet commented, did it not work?

Reply:
Oops I'll use this now, it was because originally I had included my optimization before `sugar.egg` so `NewLet` didn't exist and I didn't realize why yet.